docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -395,20 +429,24 @@ CREATE EDGE follow(degree int);
 CREATE EDGE serve(start_year int, end_year int);
 ```
 
-* IF you install **Nebula Graph** by compiling the source code, you can batch write to console by the following command:
+* If you install **Nebula Graph** by compiling the source code, you can batch write to console by the following command:
 
-```bash
+```$
```

Comment:
```bash
$ cat ...
```

Reply:
Do you mean replace bash with cat? What's the difference as this part does not expose to users.